Issue36 parse graph

JayHuLBL opened the pull request:
Remain issues:
- [ ] add name of input and output
- [ ] fix like when parsing class `CDL.Continuous.MultiSum`, where the input icon shape is changed from its original class.
- [ ] ensure in `cdl` parsing mode, can also parse package. This will be issue #47.

mwetter commented:
@JayHuLBL 
- [ ] Can you see why the CI testing fails and if needed update the reference results
- [ ] Can you address the remaining issue you list above, or put these into a separate issue. Otherwise they will disappear once the pull request is merged.

JayHuLBL commented:
@mwetter 
- [x] fixed the CI testing fails and updated the reference results
- [x] put the remain issues into a separate issue (#48 )

The SVG figure is added to html as inline. It can be downloaded by clicking the `Download` button besides it.

JayHuLBL commented:
The command `npm test` works well in local but not in the CI. I will further fix it.
